content/docs/reference/pkg/aws/cloudwatch/metricalarm/_index.md

```diff
@@ -1452,7 +1452,7 @@ The following values are supported: `ignore`, and `evaluate`.
 <a href="#insufficientdataactions_nodejs" style="color: inherit; text-decoration: inherit;">insufficient<wbr>Data<wbr>Actions</a>
 </span>
         <span class="property-indicator"></span>
-        <span class="property-type">string | Topic[]</span>
+        <span class="property-type">(string | Topic)[]</span>
```

Comment:
This looks good this is the effect of https://github.com/pulumi/pulumi/pull/8116 fix. So this auto-PR is attributed to the wrong originating PR but still seems helpful.